src/main/java/amidst/gui/profileselect/LocalProfileComponent.java

```diff
@@ -81,6 +82,13 @@ private boolean tryLoad() {
 		try {
 			Log.i("using minecraft launcher profile '" + getProfileName() + "' with versionId '" + getVersionName()
 					+ "'");
+					
+			String possibleModProfiles = "optifine|forge";
+            if (Pattern.matches(possibleModProfiles, getProfileName().toLowerCase())) {
```

Comment:
Yes that is badly tested by me, ill fix the regex
I was not aware of the locale.